modules/model/src/main/java/io/cattle/platform/core/constants/HostConstants.java

```diff
@@ -17,6 +17,7 @@
     public static final String FIELD_INSTANCE_IDS = "instanceIds";
     public static final String FIELD_IP_ADDRESS = "agentIpAddress";
     public static final String FIELD_LABELS = "labels";
+    public static final String FIELD_IMPORTED = "imported";
```

Comment:
This isn't a field so name the constant `DATA_IMPORTED` and put in a different section of the file

Reply:
To be honest the UI might want to show imported hosts differently.  So just make it a field :)